packages/jest-jasmine2/src/extendJasmineExpect.js

```diff
@@ -14,7 +14,7 @@ const jasmineExpect = global.expect;
 // extend jasmine matchers with `jest-matchers`
 global.expect = actual => {
   const jasmineMatchers = jasmineExpect(actual);
-  const jestMatchers = jestExpect(actual);
+  const jestMatchers = jestExpect(actual, global.matchersContext);
```

Comment:
can we find a way to not use a global variable to contain this data? I really would like us to avoid this.